From 2796e881ee96c66284f845f3fb196e643d1aaccb Mon Sep 17 00:00:00 2001
From: jim <jim.yuan@supermicro.com>
Date: Wed, 1 Feb 2017 15:31:21 -0800
Subject: [PATCH] print debug message if there are huge memory size request.

---
 src/kernel/pagemgr.C | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/src/kernel/pagemgr.C b/src/kernel/pagemgr.C
index c5e7d69..a50a69a 100644
--- a/src/kernel/pagemgr.C
+++ b/src/kernel/pagemgr.C
@@ -153,7 +153,8 @@ void* PageManager::allocatePage(size_t n, bool userspace)
 
     // In non-kernel mode, make a system-call to allocate in kernel-mode.
     if (!KernelMisc::in_kernel_mode())
-    {
+    { 
+    	size_t attempts = 0;
         while (NULL == page)
         {
             page = _syscall1(Systemcalls::MM_ALLOC_PAGES,
@@ -162,7 +163,13 @@ void* PageManager::allocatePage(size_t n, bool userspace)
             // Didn't successfully allocate, so yield in hopes that memory
             // will eventually free up (ex. VMM flushes).
             if (NULL == page)
-            {
+            { 
+            	attempts++;
+				if (attempts == 10000)
+					{
+						printk ("Cannot allocate %ld pages\n", n);
+					}	
+			
                 task_yield();
             }
         }
-- 
2.9.2.windows.1

