From 66380539ea7312f242bf23d9f54ff5540a75811b Mon Sep 17 00:00:00 2001
From: Oliver O'Halloran <oohall@gmail.com>
Date: Fri, 18 Jan 2019 14:33:51 +1100
Subject: [PATCH] phb4: Don't write to the IODA tables when fenced

Signed-off-by: Oliver O'Halloran <oohall@gmail.com>
---
 hw/phb4.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hw/phb4.c b/hw/phb4.c
index cd1b14c7..ec6bc305 100644
--- a/hw/phb4.c
+++ b/hw/phb4.c
@@ -154,6 +154,8 @@ static bool pci_eeh_mmio;
 static bool pci_retry_all;
 static int rx_err_max = PHB4_RX_ERR_MAX;
 
+static bool phb4_fenced(struct phb4 *p);
+
 /* Note: The "ASB" name is historical, practically this means access via
  * the XSCOM backdoor
  */
@@ -1128,6 +1130,9 @@ static int64_t phb4_ioda_reset(struct phb *phb, bool purge)
 		phb4_init_ioda_cache(p);
 	}
 
+	if (phb4_fenced(p))
+		return OPAL_HARDWARE;
+
 	/* Init_30..31 - Errata workaround, clear PESTA entry 0 */
 	phb4_ioda_sel(p, IODA3_TBL_PESTA, 0, false);
 	out_be64(p->regs + PHB_IODA_DATA0, 0);
-- 
2.21.0.windows.1

